internal/pkg/runtime/engines/singularity/cleanup_linux.go

```diff
@@ -6,12 +6,17 @@
 package singularity
```

Comment:
Missing copyright thing?

```suggestion
// Copyright (c) 2019, Sylabs Inc. All rights reserved.
// This software is licensed under a 3-clause BSD license. Please consult the
// LICENSE.md file distributed with the sources of this project regarding your
// rights to use or distribute this software.

package singularity
```

Reply:
Sorry, my bad. There already one, but it should be `2018-2019`. (not `2018`)

Reply:
Fixed